tests/test_sources.py

```diff
@@ -96,5 +96,6 @@ def test_invalid_version(mock_default_download):
         "fake version 1.0",
         "fakesdisturl.com",
     )
+    mock_default_download.__name__ = "mock_default_download"
```

Comment:
Could you push this as a separate PR, need this fix in to stop the unit tests from failing

Reply:
okay, creating a separate PR now